x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/privilege/ClusterPrivilegeResolver.java

```diff
@@ -325,6 +325,11 @@ public class ClusterPrivilegeResolver {
         CROSS_CLUSTER_REPLICATION_PATTERN
     );
 
+    public static final NamedClusterPrivilege WRITE_CONNECTOR_SECRETS = new ActionClusterPrivilege(
```

Comment:
There was, but I removed it. Originally the `read_connector_secrets` cluster privilege was being applied to the Enterprise Search service account, but because it already has `manage` it can access everything. I can't effectively limit its scope without reworking that.
So in any case, creating a cluster privilege for reads isn't necessary yet.

Reply:
I've re-added the `read_connector_secrets` cluster privilege. A similar conversation is taking place [here](https://github.com/elastic/elasticsearch/pull/103683#discussion_r1447266495) so I'll resolve this one.